server/cli/Cargo.toml

```diff
@@ -5,6 +5,7 @@ edition = "2018"
 
 [lib]
 path = "src/lib.rs"
+doctest = false
```

Comment:
is this to speed up the tests? should we do this in develop?

Reply:
I actually cherry picked this from https://github.com/openmsupply/open-msupply/pull/388/commits/27fae96d97d41a43dcce0d702ed19259cfdc0198 which is in `main`

Reply:
ok great, remembered something like this...

Try a `cargo clean` and a `rustup update` I had this error as well for a while and suspected a bad Rust version because I haven't seen it anymore...